barretenberg/cpp/src/barretenberg/vm/generated/avm_prover.cpp

```diff
@@ -974,8 +974,7 @@ void AvmProver::execute_pcs_rounds()
                                                  sumcheck_output.challenge,
                                                  commitment_key,
                                                  transcript);
-    PCS::compute_opening_proof(
-        commitment_key, prover_opening_claim.opening_pair, prover_opening_claim.polynomial, transcript);
+    PCS::compute_opening_proof(commitment_key, prover_opening_claim, transcript);
```

Comment:
This will break when regenerating. You've been modifying a generated file. (also when this was renamed from zeromorph to PCS).

The generator is here: https://github.com/AztecProtocol/aztec-packages/blob/34eb5a01d34e5ff5d1414fff53ca0623d83bde5d/bb-pilcom/bb-pil-backend/src/prover_builder.rs#L193

I know it might be a bit beyond your scope to modify the generator. In that case please do ping us so that we do it ourselves.

Reply:
apologies, would be great if you're able to modify this; also this is the last PR in which I'm tinkering with functions that might require modifying the generation for now

Reply:
I'm told that it was agreed that we'd rely on the CODEOWNERS to notify us, so nothing is needed from you. Looks like we were just a bit delayed and that I didn't know this. Apologies!